src/os/style/style.js

```diff
@@ -1251,6 +1251,10 @@ os.style.createFeatureStyle = function(feature, baseConfig, opt_layerConfig) {
 os.style.mergeConfig = function(from, to) {
   for (var key in from) {
     var fval = from[key];
+    if (fval === os.object.IGNORE_VAL) {
+      continue;
+    }
+
     if (fval && typeof fval == 'object' && !(typeof fval.length == 'number')) {
       // clone objects into the target
       if (!(key in to)) {
```

Comment:
This needs to handle `to[key]` being `null` or `undefined`. This is causing an error when hovering the polygons without a stroke, because it tries merging the highlight stroke into `null`.

Reply:
Fixed and added tests. `undefined` was already handled.